fdbserver/workloads/BackupS3BlobCorrectness.actor.cpp

```diff
@@ -167,6 +172,14 @@ struct BackupS3BlobCorrectnessWorkload : TestWorkload {
 		initSnapshotInterval = getOption(options, "initSnapshotInterval"_sr, 0);
 		snapshotInterval = getOption(options, "snapshotInterval"_sr, 30);
 
+		// Chaos testing options
+		enableChaos = getOption(options, "enableChaos"_sr, false);
+		errorRate = getOption(options, "errorRate"_sr, 0.0);
```

Comment:
What's the behavior of `S3FaultInjector::injector` when it is fed with  the default 0.0 values?

Reply:
If enableChaos is true and injector values are 0.0, the mocks3chaos server acts like the mocks3 server (which is run when enableChaos is false) and there is no chaos: no errors, etc.